app/components/object-inspector.hbs

```diff
@@ -2,43 +2,52 @@
   <div class="split__panel__hd">
     <div class="toolbar">
       <button
-        class="toolbar__icon-button {{if this.isNested "enabled" "disabled"}} js-object-inspector-back"
+        class="toolbar__icon-button {{if this.isNested "enabled" "disabled"}}"
         {{on "click" this.popStack}}
+        data-test-object-inspector-back
       >
         {{svg-jar "arrow-back" width="21px" height="21px"}}
       </button>
 
-      <Ui::ToolbarDivider />
```

Comment:
We no longer need this divider?

Reply:
To me it seems like extra noise.

<img width="123" alt="image" src="https://user-images.githubusercontent.com/3692/78506889-b7dc6a80-7731-11ea-8549-506f928086f2.png">

<img width="114" alt="image" src="https://user-images.githubusercontent.com/3692/78506916-e5c1af00-7731-11ea-8949-274f20232a98.png">

<img width="177" alt="image" src="https://user-images.githubusercontent.com/3692/78506970-36d1a300-7732-11ea-8fda-ee6ef552a311.png">

<img width="151" alt="image" src="https://user-images.githubusercontent.com/3692/78506972-3a652a00-7732-11ea-9038-19a23db1cecf.png">